docs/development.md

```diff
@@ -0,0 +1,153 @@
+# Development
+
+Like a lot of other projects at Hackney, this project uses Next.js written in Javascript. This means you'll need a familiarality with
+
+* [Next.JS](https://nextjs.org/)
+* [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+* [React](https://reactjs.org/)
+
+## Architecture
+
+The diagram below shows the overall data flow between the various moving components for Better Conversations.
+
+![Data Flows](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/LBHackney-IT/coronavirus-frontdoor-snapshot/master/docs/Data-Flow.iuml)
+
+### AWS Accounts 
+
+This project is hosted in the Hackney AWS accounts, depending on the stage you're looking for.
+
+* Production: `ProductionAPIs`
+* Staging: `StagingAPIs`
+
+![AWS Architecture](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/LBHackney-IT/coronavirus-frontdoor-snapshot/master/docs/infrastructure.iuml)
```

Comment:
Are we .iuml's supposed to show up as images?

![image](https://user-images.githubusercontent.com/43747286/112476786-2f077600-8d6a-11eb-93a1-3558f104c795.png)

Reply:
Please see updated description. The URL for the iuml file doesn't yet exist (as it's not merged in yet)